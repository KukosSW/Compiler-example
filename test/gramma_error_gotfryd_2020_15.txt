DECLARE
	c
BEGIN
	c := 0;
	FOR i FROM 1 TO i DO
		c := c + i;
	ENDFOR

	WRITE c;

END