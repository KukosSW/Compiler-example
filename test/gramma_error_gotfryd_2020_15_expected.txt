BLAD: Niezadeklarowana zmienna i w linii 5
